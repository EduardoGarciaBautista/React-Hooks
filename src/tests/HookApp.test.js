import { shallow } from 'enzyme';
import React from 'react';
import '@testing-library/jest-dom';
import { HookApp } from '../HookApp';

describe('Hook App test', () => {
    test('should be equals to snapshot', () => {
        const wrapper = shallow(<HookApp/>);

        expect(wrapper).toMatchSnapshot();
    });
    
})