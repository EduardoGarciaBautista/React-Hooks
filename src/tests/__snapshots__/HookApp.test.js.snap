// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hook App test should be equals to snapshot 1`] = `
<h1>
  Hola
</h1>
`;
